app/assets/src/components/Samples.jsx

```diff
@@ -588,29 +587,14 @@ class Samples extends React.Component {
       return (
         <a className="col s12 no-padding sample-feed" key={i}>
           <div>
-            <div className="samples-card white">
-              <div className="flex-container">
-                <ul className="flex-items">
-                  <li className="check-box-container">
-                    {this.state.displaySelectSamples ? (
-                      <div>
-                        <input
-                          type="checkbox"
-                          data-sample-id={dbSample.id}
-                          id={i}
-                          onClick={this.selectSample}
-                          className="filled-in checkbox"
-                          value={
-                            this.state.selectedSampleIds.indexOf(dbSample.id) !=
-                            -1
-                          }
-                          disabled={status != "COMPLETE"}
-                        />{" "}
-                        <label htmlFor={i}>{sample_name_info}</label>
-                      </div>
-                    ) : (
-                      sample_name_info
-                    )}
+            <div className='samples-card white'>
+              <div className='flex-container'>
+                <ul className='flex-items'>
+                  <li className='check-box-container'>
+                    { this.state.displaySelectSamples ? <div><input type="checkbox" id={i} onClick = { this.selectSample } key= {`sample_${dbSample.id}`} data-sample-id ={dbSample.id}
```

Comment:
If statements of this complexity are generally discouraged to do inline.

Reply:
yep. we have tons of these.